packages/components/src/tree-grid/README.md

```diff
@@ -108,10 +108,33 @@ function TreeMenu() {
 
 ##### Props
 
-`TreeGrid` accepts no specific props. Any props specified will be passed to the `table` element rendered by `TreeGrid`.
+Aside from the documented callback functions, any props specified will be passed to the `table` element rendered by `TreeGrid`.
 
 `TreeGrid` should always have children.
 
+###### onChangeRow( event: Event, startRow: HTMLElement, destinationRow: HTMLElement )
```

Comment:
Usually `onChange` is associated more to a value change for a input-like element — maybe `onFocusRow` is a better name?

Reply:
Oh, great idea, that's a better fit. Updated in https://github.com/WordPress/gutenberg/pull/38314/commits/5511896de7cc60499a24ffece4cefb64c68b1b15